src/App.js

```diff
@@ -1,26 +1,67 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React,  {useState} from 'react';
+import Foods from './foods.json';
+import 'bulma/css/bulma.css';
+import FoodBox from './Components/FoodBox';
+import FoodForm from './Components/FoodForm';
+import SearchBar from './Components/SearchBar';
+import Menu from './Components/Menu.js';
 
 function App() {
+  const [foodList, setFood] = useState(Foods);
+  const [formVisible, setFormVisible] = useState("");
+  const [foodForm, setFoodForm] = useState({
+    name: '',
+    calories: 0,
+    image: '',
+  });
+  const [todayFood, setTodayFood] = useState([  ]);
+
+  function handleAddFood(e) {
+    e.preventDefault();
+    setFormVisible('is-active');
+  };
+
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div className="container">
+      <h1 className="title is-1">Iron Nutrition</h1>
+        <SearchBar listFood={[Foods, setFood]} />
+        <div className="columns">
+        <div className="column is-three-quarters">
+          {foodList.map((food, index) => {
+            return (
+              <FoodBox
+                name={food.name}
+                calories={food.calories}
+                image={food.image}
+                quantity={food.quantity}
+                key={index}
+                keyId={index}
+                todayFood={[todayFood, setTodayFood]}
+                foodList={[foodList, setFood]}
+              />
+            );
+          })}
+        </div>
+        <div className="column is-one-quarter">
+          <Menu>{[todayFood, setTodayFood]}</Menu>
+          <h3 className="title is-3">Add new Food</h3>
+          <button className="button is-primary" onClick={handleAddFood}>
+            Add a Food
+          </button>
+
```

Comment:
I just wanted to mention an alternative way to use conditional rendering for line 52-59, it's called Short Circuit Evaluation( number 5 of [this article](https://www.digitalocean.com/community/tutorials/7-ways-to-implement-conditional-rendering-in-react-applications)).

It will come in handy to know how to use it.
```
{formVisible && <FoodForm
              state={[foodForm, setFoodForm]}
              stateFoods={[foodList, setFood]}
              stateVisible={[formVisible, setFormVisible]}
            /> }